main.py

```diff
@@ -3,6 +3,10 @@
 import os, time, re
 import threading
 import json
+import gi
```

Comment:
this is optional in case you wanna work with a gui in the future... again the tkinter one was just an option you don't need to do it strictly :)